server/libs/infra/src/migrations/1681144628393-AddOriginalFileNameToAssetTable.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddOriginalFileNameToAssetTable1681144628393 implements MigrationInterface {
+    name = 'AddOriginalFileNameToAssetTable1681144628393'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "assets" ADD "originalFileName" character varying`);
```

Comment:
We can probably do a migration here to update original filename from the exif table.